packages/visualizations-react/stories/Chart/RadialCharts/RadialCharts.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import type { ChartOptions, DataFrame } from '@opendatasoft/visualizations';
+import { ChartSeriesType } from '@opendatasoft/visualizations';
+import { Meta } from '@storybook/react';
+import type { Props } from '../../../src';
+
+import ChartTemplate from '../ChartTemplate';
+
+const meta: Meta = {
+    title: 'Chart/RadialCharts',
+};
+
+export default meta;
+
+export const PieChart = ChartTemplate.bind({});
+const PieChartArgs: Props<DataFrame, ChartOptions> = {
+    data: {
+        loading: false,
+        value: [
+            { series_001: 17,  x: 'Acer'},
+            { series_001: 22,  x: 'Aesculus' },
+            { series_001: 18,  x: 'Alnus' },
+        ],
+    },
+    options: {
+        labelColumn: 'x',
+        title: {
+            text: 'Pie Chart',
+        },
+        series: [
+            {
+                type: ChartSeriesType.Pie,
+                valueColumn: 'series_001',
+                backgroundColor : [
+                    "#CB4335",
+                    "#1F618D",
+                    "#F1C40F",
+                    ]
+            },
+        ],
+    },
+};
+PieChart.args = PieChartArgs;
+
+export const DoughnutChart = ChartTemplate.bind({});
+const DoughnutChartArgs: Props<DataFrame, ChartOptions> = {
+    data: {
+        loading: false,
+        value: [
+            { series_001: 17,  x: 'Acer'},
+            { series_001: 22,  x: 'Aesculus' },
+            { series_001: 18,  x: 'Alnus' },
+        ],
+    },
+    options: {
+        labelColumn: 'x',
+        title: {
+            text: 'Pie Chart',
+        },
+        series: [
+            {
+                type: ChartSeriesType.Pie,
```

Comment:
IMO, we need to set the type to `ChartSeriesType.Doughnut` so that even if the cutout option is not defined the chart will have a default cutout of '50%'

https://www.chartjs.org/docs/latest/charts/doughnut.html#doughnut-and-pie-charts

Reply:
You're right, and then the `cutout` option belongs to the serie which makes more sense => if we have a `Doughnut` serie then we have a cutout in this serie, else we don't